src/Apps/Order/Routes/Accept/index.tsx

```diff
@@ -1,7 +1,6 @@
-import { Button, Col, Flex, Row, Spacer } from "@artsy/palette"
+import { Box, Button, Col, Flex, Row, Spacer } from "@artsy/palette"
 import { Accept_order } from "__generated__/Accept_order.graphql"
 import { HorizontalPadding } from "Apps/Components/HorizontalPadding"
-import { Helper } from "Apps/Order/Components/Helper"
```

Comment:
Is this `Helper` component used anywhere else? We may be able to just remove it entirely.